ui/threejs/AnchorControls.js

```diff
@@ -233,134 +186,144 @@ class AnchorControls extends EventDispatcher {
     const onKeyDown = (event) => {
       if (!_object) return;
 
-      switch (event.key) {
-        // step
-        case '1':
-          _step = 100.0;
-          break;
-        case '2':
-          _step = 50.0;
-          break;
-        case '3':
-          _step = 25.0;
-          break;
-        case '4':
-          _step = 10.0;
-          break;
-        case '5':
-          _step = 5.0;
-          break;
-        case '6':
-          _step = 2.5;
-          break;
-        case '7':
-          _step = 1.0;
-          break;
-        case '8':
-          _step = 0.5;
-          break;
-        case '9':
-          _step = 0.25;
-          break;
-        case '0':
-          _step = 0.1;
-          break;
-
-        // at
-        case 'd':
-          _at.position.addScaledVector(_xAxis, _step);
-          if (_lockUp) {
-            _up.position.addScaledVector(_xAxis, _step);
-          }
-          if (_lockTo) {
+      if (event.getModifierState('Control')) {
```

Comment:
Mac users might expect this to be apple+C...but I'm comfortable with that not working

Reply:
I think apple has the "Copy" key, etc, but I might be confused.

But we can add those in.